BibBuddy/app/src/main/java/de/bibbuddy/LibraryFragment.java

```diff
@@ -63,6 +64,7 @@ public void handleOnBackPressed() {
 
     setHasOptionsMenu(true);
     selectedShelfItems = new ArrayList<ShelfItem>();
+    selectedBookItems = new ArrayList<BookItem>();
```

Comment:
Why do you create a selectedBookItems List in the LibraryFragment? In the library you can't select a book. You can only select shelves.